### ISSUE TITLE
Docs Missing over the last week?

### ISSUE THREAD

**Issue by dvetsch75** [pyjanitor-devs/pyjanitor#858]
I might be wrong, but I think this was fully documented last week. But today, when I navigate to the docs page, I see this:
![image](https://user-images.githubusercontent.com/70990721/127163336-be235ab4-a0b4-4744-a8be-ad2f0d1945b4.png)

Two questions on this:
 1. Am I going crazy?
 2. If I am not going crazy, is it possible to revert to the docs that I saw last week?

**Comment by ericmjl** [comment]
You're not crazy, @dvetsch75!

I thought I was crazy when I saw the docs on readthedocs missing.

However, I checked in with the dev team later and they helped me remember that our official docs are at: https://pyjanitor-devs.github.io/pyjanitor/.

Last week, I think @loganthomas or @jk3587 updated the URL on the repo to reflect the change.

Hope this helps!

I think we might be taking down the readthedocs site until we can get it working properly.
